sql/updates/world/2024_14_08_world_broken_shore_scenario.sql

```diff
@@ -0,0 +1,16 @@
+-- remove second LFG queue, already gets triggered by end of scene
+-- INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (227058, 'spell_q42740');
+DELETE FROM spell_script_names WHERE spell_id = 227058 AND ScriptName = 'spell_q42740';
+
+-- disable custom event (modified uwow Legion Invasion - Westfall/Barrens)
+UPDATE game_event SET start_time = '0000-00-00 00:00:00', end_time = '0000-00-00 00:00:00', length = '1' WHERE eventEntry = 811;
```

Comment:
I suggest use 2000-01-01 00:00:00 for start_time and 2048-01-01 00:00:00 for end_time. '0000-00-00 00:00:00' is not vaild for some mysql sqlmode.

Reply:
I know , we need change date and time

Reply:
If we change this , works good